gluster-exporter/main.go

```diff
@@ -120,6 +120,12 @@ func main() {
 
 	for _, m := range glusterMetrics {
 		if collectorConf, ok := exporterConf.CollectorsConf[m.name]; ok {
+			if collectorConf.Name == glusterutils.GlusterVolumeProfile && !collectorConf.Disabled {
```

Comment:
Sorry for missing this earlier. If we enable profiling only during exporter starts then all new Volume will not have this option enabled which are created after exporter starts. I think we have to move this enable thing while collecting profile itself. (Only when volinfo doesn't have this enabled). Function `EnableVolumeProfiling` should accept volinfo and enable one volume profiling at a time.

Reply:
Ack. Will move this code accordingly.